Makefile

```diff
@@ -97,15 +96,6 @@ check-go-format: prebuild-check
 	|| true
 
 
-
-.PHONY: release
-release: all
```

Comment:
The goRelease {} function needs to call `make -e BRANCH=master release` so you will need this release `release: all` target there but if you don't want the github release too then omit the next 5 lines below.  This should then build and push your image to dockerhub